freqtrade/freqai/freqai_interface.py

```diff
@@ -205,9 +205,13 @@ def _start_scanning(self, strategy: IStrategy) -> None:
 
             if retrain:
                 self.train_timer('start')
-                self.extract_data_and_train_model(
-                    new_trained_timerange, pair, strategy, dk, data_load_timerange
-                )
+                try:
+                    self.extract_data_and_train_model(
+                        new_trained_timerange, pair, strategy, dk, data_load_timerange
+                    )
+                except Exception as msg:
+                    logger.warning(f'Training {pair} raised exception {msg}, skipping.')
```

Comment:
i think you need to be careful with logging.
this way, you'll only get the exception type (for example ValueError) - but not the actual explanation.

not necessarily a problem - but can hinder actual debugging / error search.

Reply:
This exception is hard to predict - could arise from any library the user brings in. I guess we could try to wrap this similar to the strategy wrapper so we can get slightly more detailed exceptions?